### PR TITLE
Fix partial issues from #10198

### PR DESCRIPTION
**Pull request by Honry**
- `if (testRunner)` should be `if (window.testRunner)`
 - Use String to pass sensor object param to avoid undefined error

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/10962)

### CODE REVIEW

**Comment by Honry** [comment]
@pozdnyakov, @alexshalamov, PTAL.

**Comment by Honry** [comment]
@foolip @rakuco, could you help review this PR? I am going to bring more methods from Chromium sensor mojo interfaces to WPT. e.g. `setUpdateSensorReadingFunction`, `setStartShouldFail`, etc., which is somehow based on this PR.

File: resources/chromium/generic_sensor_mocks.js [hunk 1]
```diff
@@ -201,10 +201,10 @@ var GenericSensorTest = (() => {
       if (testInternal.initialized)
         throw new Error('Call reset() before initialize().');
 
-      if (testRunner) { // Grant sensor permissions for Chromium testrunner.
+      if (window.testRunner) { // Grant sensor permissions for Chromium testrunner.
         ['accelerometer', 'gyroscope',
          'magnetometer', 'ambient-light-sensor'].forEach((entry) => {
-          testRunner.setPermission(entry, 'granted',
+          window.testRunner.setPermission(entry, 'granted',
                                    location.origin, location.origin);
```
Comment (rakuco) [inline resources/chromium/generic_sensor_mocks.js]: You need to reindent this line as well.

File: resources/chromium/generic_sensor_mocks.js [hunk 2]
```diff
@@ -201,10 +201,10 @@ var GenericSensorTest = (() => {
       if (testInternal.initialized)
         throw new Error('Call reset() before initialize().');
 
-      if (testRunner) { // Grant sensor permissions for Chromium testrunner.
+      if (window.testRunner) { // Grant sensor permissions for Chromium testrunner.
```
Comment (rakuco) [inline resources/chromium/generic_sensor_mocks.js]: Sorry if this sounds dumb, but isn't `testRunner` the same as `window.testRunner` here? Or do the Mojo JS files behave differently?

Comment (Honry) [reply resources/chromium/generic_sensor_mocks.js]: This `testRunner` is specific for Chromium testrunner, not sure if it is still useful in `LayoutTests/external/wpt`, but for sure, it is `undefined` in w-p-t runnner.

Comment (Honry) [reply resources/chromium/generic_sensor_mocks.js]: i.e. tests will fail with `Uncaught ReferenceError: testRunner is not defined`

File: resources/chromium/generic_sensor_mocks.js (hunk 1 above)
Comment (Honry) [reply resources/chromium/generic_sensor_mocks.js]: thanks! Fixed.

**Review by foolip — CHANGES_REQUESTED** [review]
Aren't all the tests failing now? run_fp_tests_* and runGenericSensor* need to updated, right?

**Comment by Honry** [comment]
@foolip, most tests are still failing, for the reason, please see my comment at https://github.com/web-platform-tests/wpt/issues/10906#issuecomment-388394993

**Review by rakuco — APPROVED** [review]
The changes look fine to me, even if there might be additional work left. @foolip?

**Comment by foolip** [comment]
@Honry, even if the tests are already failing, won't these changes make them further from passing? If they're already passing in Chromium, could you try making the changes in a Chromium CL and see if they stay passing?

**Comment by rakuco** [comment]
@foolip AFAICS, this PR should get these tests closer, not further, to passing. `ambient-light/` and `proximity` are currently not imported into Chromium, and `geolocation-sensor/` is failing there due to `GeolocationSensor` not being implemented at all in Blink.

Other directories which call those functions (e.g. `magnetometer/`) are already using these function call forms.

**Comment by foolip** [comment]
Hmm, something is very strange, run_fp_tests_disabled and runGenericSensorTests already assumes a string argument, but some tests still pass interface objects. So yes, these tests are moving things in the right direction. (I assumed that the helpers weren't updated, in which case this made no sense.)

**Review by foolip — APPROVED** [review]

**Pull request merged**